authentik/sources/ldap/tests/test_sync.py

```diff
@@ -308,3 +314,124 @@ def test_tasks_openldap(self):
         connection = MagicMock(return_value=mock_slapd_connection(LDAP_PASSWORD))
         with patch("authentik.sources.ldap.models.LDAPSource.connection", connection):
             ldap_sync_all.delay().get()
+
```

Comment:
Can we have a tests that makes sure that if the user exists at the source, it doesn't get deleted?

I suppose the ones above do implicitly test that though